app/views/two_factor_authentication/otp_expired/show.html.erb

```diff
@@ -0,0 +1,51 @@
+<% title t('titles.otp_expired') %>
+
+<%= render AlertIconComponent.new(icon_name: :error, class: 'display-block margin-bottom-4') %>
+<%= render PageHeadingComponent.new.with_content(t('headings.otp_expired')) %>
+
+  <%= render ButtonComponent.new(
+        action: ->(**tag_options, &block) do
+          link_to(
+            otp_send_path(
+              otp_delivery_selection_form: {
+                otp_delivery_preference: @otp_delivery_preference,
+                resend: true,
+              },
+            ),
+            **tag_options,
+            &block
+          )
+        end,
+        outline: true,
+        icon: :loop,
+        class: 'margin-bottom-neg-1',
+      ).with_content(t('links.two_factor_authentication.send_another_code')) %>
+
+<%= render(
+      'shared/troubleshooting_options',
+      heading_tag: :h2,
+      heading: t('components.troubleshooting_options.default_heading'),
+      options: [
+        {
+          url: login_two_factor_options_url,
+          text: t('forms.two_factor.try_again'),
+        },
+        {
+          url: login_two_factor_options_url,
+          text: t('two_factor_authentication.login_options_link_text'),
+        },
+        {
+          url: help_center_redirect_path(
+            category: 'verify-your-identity',
+            article: 'phone-number-and-phone-plan-in-your-name',
+          ),
+          text: t('links.contact_support', app_name: APP_NAME),
+          new_tab: true,
+        },
+        @sp_name && {
+          url: idv_doc_auth_return_to_sp_path(location: 'missing_items'),
+          text: t('idv.troubleshooting.options.get_help_at_sp', sp_name: decorated_session.sp_name),
+          new_tab: true,
+        },
```

Comment:
Can we double-check these troubleshooting options? I wouldn't expect to see troubleshooting options related to identity verification here.